docs/topics/testing/overview.txt

```diff
@@ -340,15 +340,28 @@ gain a speed up on multi-core hardware. See :option:`test --parallel`.
 Password hashing
 ~~~~~~~~~~~~~~~~
 
-The default password hasher is rather slow by design. If you're authenticating
-many users in your tests, you may want to use a custom settings file and set
-the :setting:`PASSWORD_HASHERS` setting to a faster hashing algorithm::
+The default password hasher is rather slow by design so when running tests,
+Django changes the :setting:`PASSWORD_HASHERS` setting to a faster algorithm,
+which is equivalent to running tests with::
 
     PASSWORD_HASHERS = [
-        'django.contrib.auth.hashers.MD5PasswordHasher',
+        'django.contrib.auth.hashers.UnsaltedSHA1PasswordHasher',
     ]
 
-Don't forget to also include in :setting:`PASSWORD_HASHERS` any hashing
+.. versionadded:: 3.1
+
+    This behavior was automated via the :setting:`TEST_PASSWORD_HASHER` setting.
+
+To opt-out from this behavior, you can set the :setting:`TEST_PASSWORD_HASHER`
+setting to `None`, which would leave your :setting:`PASSWORD_HASHERS` setting unchanged::
+
+    TEST_PASSWORD_HASHER=None
```

Comment:
Can you add spacing between the equals here and below?

Reply:
Ah yes of course! Updated.